core/src/consensus.rs

```diff
@@ -1619,583 +1617,6 @@ pub mod test {
         );
     }
 
-    #[test]
```

Comment:
Why are we removing all the tower tests from here? This is where we define the tower, I think the tests should remain here.